src/templates/cart/shopping_cart.template.html

```diff
@@ -261,7 +261,7 @@ <h4 class="panel-title">Discount Code</h4>
 					<td class="text-right" ><h3>[%format type:'currency'%][@grand_total@][%/format%]</h3></td>
 				</tr>
 			</table>
-			<form method="post"  name="paypal" action="[%url page:'checkout' fn:'3rdparty'/%]">
+			<form method="post"	name="paypal" action="[%url page:'checkout' fn:'3rdparty'/%]">
```

Comment:
Looks like there were two spaces here, and you did a find and replace to replace that with tabs to fix indenting, and it's now put tabs between the HTML attributes. I've seen it in a few places.